context/AppContext.tsx

```diff
@@ -0,0 +1,2 @@
+import { createContext } from 'react';
+export default createContext<string>("");
```

Comment:
`AppContext` doesn't have string data type. It is a object that contains `path` variable in it.

Reference - https://github.com/asyncapi/website/blob/master/pages/_app.js#L31